ejb/WFLY-13828-remote+tls-is-not-supported-by-EJBClient-and-remote-outbound-connection.adoc

```diff
@@ -0,0 +1,157 @@
+---
+categories:
+  - ejb
+  - remoting
+stability-level: default
+issue: https://issues.redhat.com/browse/WFLY-13828
+feature-team:
+ developer: tadamski
+ sme:
+  - tadamski
+ outside-perspective:
+  - mkopecky
+promotes:
+promoted-by:
+= remote+tls is not supported by EJBClient and remote-outbound-connection
```

Comment:
You need to add `---` before L15 as a delimiter for the yaml markup

Reply:
added, looking at current bug